ec2/sc-portfolio-ec2.yaml

```diff
@@ -137,7 +142,7 @@ Resources:
           - !Sub 'arn:aws:iam::${AWS::AccountId}:role/${LaunchRoleName}'
         PortfolioId: !Ref 'SCEC2portfolio'
         RepoRootURL: !Ref 'RepoRootURL'
-        ProductVersionName: !Ref 'ProductVersionName'
+        StackDatetime: !Ref StackDatetime
```

Comment:
no longer using `ProductVersionName`?  maybe remove it from parameters list? you may need to update tests as well.